ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineNewPayload.java

```diff
@@ -184,16 +184,15 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext)
           transaction ->
               mergeCoordinator
                   .getEthScheduler()
-                  .scheduleFutureTask(
+                  .scheduleTxWorkerTask(
```

Comment:
last suggestions, sync txWorker by default has a limit of 2 threads, so it could not leverage all the cores, for that you can change the number of thread to reflect the number of cores, or use the services executor that uses a cached thread executor with not predefined limit on threads

Reply:
This method is not using the sync txWorker but the txWorker, which is a thread pool of a minimum 1 and a maximum number of threads 1000.